util/include/OpenTracing.hpp

```diff
@@ -30,21 +31,49 @@ class SpanWrapper {
   SpanWrapper& operator=(const SpanWrapper&) = delete;
   SpanWrapper(SpanWrapper&&) = default;
   SpanWrapper& operator=(SpanWrapper&&) = default;
+  ~SpanWrapper() = default;
+  explicit operator bool() const {
+#ifdef USE_OPENTRACING
+    return (impl().get() != nullptr);
+#else
+    return false;
+#endif
+  }
+
+  template <typename ValueT>
+  void setTag(const std::string& name, const ValueT& value) {
+#ifdef USE_OPENTRACING
```

Comment:
As this is not included in the Travis CI matrix, I suppose you have tested both cases at least locally?

Reply:
Yes, I did not want to add more to the matrix.
The compilation key is tested locally.